docs/src/whatsnew/index.rst

```diff
@@ -6,11 +6,15 @@
 What's New in Iris
 ------------------
 
-.. include:: 3.14.rst
+.. Commented out temporarily because Sphinx can't cope with a file that includes
+   labels being referenced twice.
+   include:: 3.14.rst
+   toctree::
+   :maxdepth: 1
+   :hidden:
```

Comment:
I've left a lot unsaid here. On release branches we reference the file - in this case `3.14.rst` - under both `include::` and under `toctree::`. That is what causes the file to be referenced twice, which means duplicating labels such as `.. _3_14_dependencies:` and Sphinx doesn't like that.

This is not a problem on `main` because `include::` is `latest.rst`, so every file is only referenced once.

So I am proposing commenting this out in the release branch, while `main` still behaves as normal.

You can see what the index page looks like under this change: https://scitools-iris--6817.org.readthedocs.build/en/6817/whatsnew/index.html

In future we can have a go with Towncrier instead and stop experiencing these problems altogether.